crates/rv/src/commands/ruby.rs

```diff
@@ -76,7 +76,7 @@ pub enum RubyCommand {
         no_install: bool,
 
         /// Ruby version to run
-        version: RubyRequest,
+        version: Option<String>,
```

Comment:
My understanding is that if we keep it to `Option<RubyRequest>` we could generate a better error message early in case the user types a version that's malformed. Though RubyRequest may need some extra updates to make it work better for it.

Reply:
OK, fixed in https://github.com/spinel-coop/rv/pull/241 which should hopefully result in nice errors everywhere a CLI command takes a `RubyRequest`.

Reply:
Updated as mentioned!